unreleased_history/behavior_changes/buffered_io_compaction_readahead_size_zero.md

```diff
@@ -0,0 +1 @@
+As `Options::access_hint_on_compaction_start` is made to have no effect since #11658, under non-direct IO, setting `Options::compaction_readahead_size` to 0 may regress compaction read performance, because it will not have the readahead issued by Kernel, which is the behavior prior to #11658, nor the intended replacement of Kernel-issued readahead by RocksDB-issued readahead introduced in #11631 now requiring `Options::compaction_readahead_size > 0`
```

Comment:
How about "Explicitly setting `Options::compaction_readahead_size` to 0 will now regress compaction read performance"?

Reply:
Fixed